packages/contracts/contracts/test/OVM_L1BlockNumber.t.sol

```diff
@@ -0,0 +1,25 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import { DSTest } from "forge-std/Test.sol";
+import { Vm } from "forge-std/Vm.sol";
+import { L1Block } from "../L2/L1Block.sol";
+import { OVM_L1BlockNumber } from "../L2/predeploys/OVM_L1BlockNumber.sol";
+
```

Comment:
If you do this:

```solidity
import { Test } from "forge-std/Test.sol";
```

And extend using `Test` instead of `DSTest`, then you don't need to worry about `Vm`. `DSTest` is the most legacy thing made by makerdao, `Test` extends `DSTest` with some nice to haves